pulp_rpm/app/viewsets.py

```diff
@@ -115,15 +117,169 @@ class PackageViewSet(SingleArtifactContentUploadViewSet):
     minimal_serializer_class = MinimalPackageSerializer
     filterset_class = PackageFilter
 
+    # TODO: adjust this policy after upload access policy design done and in place
```

Comment:
Is there an issue #?

Reply:
it is not, I removed `TODO` as endpoint is secured by rbac as it is possible right now.